src/Rector/Return_/CommandConstantReturnCodeRector.php

```diff
@@ -65,7 +65,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
      */
     public function getNodeTypes(): array
     {
-        return [Return_::class];
+        return [ClassMethod::class];
```

Comment:
namespace must be changed to under `namespace Rector\Symfony\Rector\ClassMethod`

Reply:
Fixed